### ISSUE TITLE
3DPW_validation_crowd_hhrnet_result.json

### ISSUE THREAD

**Issue by zhLawliet** [hongsukchoi/3DCrowdNet_RELEASE#22]
i can‘t find the  3DPW_validation_crowd_hhrnet_result.json
<img width="355" alt="image" src="https://user-images.githubusercontent.com/17424385/200732710-6e905a55-c259-4ea4-b04e-9b03b51d055c.png">

**Comment by hongsukchoi** [comment]
The 3dpw_validation_crowd_hhrnet_result.json is in the 2DPose_Detection folder.

I didn't prepare the MuPoTs evaluation code in this repo, since it has few crowd scenarios and is not a in-the-wild dataset.

**Comment by zhLawliet** [comment]
thanks, can you&nbsp; share the&nbsp; J_regressor_mi_smpl.npy or&nbsp; if i&nbsp; want&nbsp; to&nbsp; compare&nbsp; the value of mupots with you in my paper ，can i&nbsp; use J_regress_h36m.py&nbsp; instead?



---Original---
From: "Hongsuk ***@***.***&gt;
Date: Thu, Nov 10, 2022 02:56 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hongsukchoi/3DCrowdNet_RELEASE]3DPW_validation_crowd_hhrnet_result.json (Issue #22)




 
The 3dpw_validation_crowd_hhrnet_result.json is in the 2DPose_Detection folder.
 
I didn't prepare the MuPoTs evaluation code in this repo, since it has few crowd scenarios and is not a in-the-wild dataset.

**Comment by hongsukchoi** [comment]
I can share it. I uploaded to the joint_regressor folder.

**Comment by zhLawliet** [comment]
thanks



---Original---
From: "Hongsuk ***@***.***&gt;
Date: Thu, Nov 10, 2022 06:26 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hongsukchoi/3DCrowdNet_RELEASE]3DPW_validation_crowd_hhrnet_result.json (Issue #22)




 
I can share it. I uploaded to the joint_regressor folder.

**Comment by zhLawliet** [comment]
@hongsukchoi  can you share the MuPoTs_test_hhrnet_result.json and  MuPoTs_test_openpose_result.json, i want to reproduce the  benchmark of MuPoTs, thanks

**Comment by hongsukchoi** [comment]
It takes some time to find them.

**Comment by hongsukchoi** [comment]
And if you use GT, PCK should be much higher. Does 0.72376 mean 72.376 PCK?

**Comment by zhLawliet** [comment]
yes, 72.376 PCK



---Original---
From: "Hongsuk ***@***.***&gt;
Date: Thu, Nov 17, 2022 02:42 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hongsukchoi/3DCrowdNet_RELEASE]3DPW_validation_crowd_hhrnet_result.json (Issue #22)




 
And if you use GT, PCK should be much higher. Does 0.72376 mean 72.376 PCK?

**Comment by hongsukchoi** [comment]
Here is the hhrnet result. I cannot find the openpose result :( But I remember using this repo: https://github.com/tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation

https://drive.google.com/drive/folders/1_Xrtd6k8sFv8FHh7NOg4B8pVBX-vyzS0?usp=sharing

**Comment by zhLawliet** [comment]
thanks



---Original---
From: "Hongsuk ***@***.***&gt;
Date: Tue, Nov 22, 2022 05:25 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hongsukchoi/3DCrowdNet_RELEASE]3DPW_validation_crowd_hhrnet_result.json (Issue #22)




 
Here is the hhrnet result. I cannot find the openpose result :( But I remember using this repo: https://github.com/tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation
 
https://drive.google.com/drive/folders/1_Xrtd6k8sFv8FHh7NOg4B8pVBX-vyzS0?usp=sharing

**Comment by hongsukchoi** [comment]
When you run openpose, there could multiple outputs in the scene or even for one person. Filter them out with this code: https://github.com/hongsukchoi/3DCrowdNet_RELEASE/blob/main/tool/match_mupots_2dpose.py

**Comment by zhLawliet** [comment]
@hongsukchoi  thanks，i have done it， i find the 3DPCK of MPJPE is just 61.2. my eval code is https://github.com/ddddwee1/MuPoTS3D-Evaluation/blob/master/util/evaluate.py
<img width="952" alt="image" src="https://user-images.githubusercontent.com/17424385/203216338-b64779e8-0e29-4083-b209-77b216e6901f.png">

**Comment by hongsukchoi** [comment]
Hi,

I think there are some bugs. 
1. Use the matlab code from the original paper
2. Check the output by comparing with the input image

**Comment by zhLawliet** [comment]
ok, i try it, if i want to get the results of paper,  the chekpoint is exp_04-06_23_43 epoch10 of pretrained_3DCrowdNet?

**Comment by hongsukchoi** [comment]
Yes. I haven’t checked it before the release, but it will at least give similar results. If there’s no bug and you use detected 2d poses and gt, pck will be at least over 70 and 80 respectively. 

Also, compare the joint order with the MuPoTs dataset during debugging.

**Comment by zhLawliet** [comment]
> 

yes, you are right, i can get  72.6  for hrnet,  82.6 for GT
https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/master/data/MuPoTS/mpii_mupots_multiperson_eval.m

<img width="1159" alt="image" src="https://user-images.githubusercontent.com/17424385/203349079-0d0c9d7a-91ea-47cf-8348-b6d7653615fb.png">

**Comment by hongsukchoi** [comment]
Great! The released pre-trained weights are for reproducing 3DPW-Crowd and 3DPW, which had the fastest convergence. If you want to reproduce the exact numbers or even higher accuracy, you can train longer. Like lr_decay: [40], end_epoch: 50

**Comment by zhLawliet** [comment]
ok，thank you



---Original---
From: "Hongsuk ***@***.***&gt;
Date: Wed, Nov 23, 2022 01:02 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [hongsukchoi/3DCrowdNet_RELEASE]3DPW_validation_crowd_hhrnet_result.json (Issue #22)




 
Great! The released pre-trained weights are for reproducing 3DPW-Crowd and 3DPW, which had the fastest convergence. If you want to reproduce the exact numbers or even higher accuracy, you can train longer. Like lr_decay: [40], end_epoch: 50
